Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -457,6 +457,7 @@ class AppSettingsTableViewController: SettingsTableViewController,
             ChangeToChinaSetting(settings: self),
             AppReviewPromptSetting(settings: self, settingsDelegate: self),
             TogglePullToRefresh(settings: self, settingsDelegate: self),
+            SwitchFakespotProduction(settings: self, settingsDelegate: self),
```

Comment:
nit, we can order the debug settings by alphabetical order (not by the setting name, but the string it shows in the actual debug setting when the app is ran). Since you have an `S` for `Switch FakespotEndpoint [...]`, it would go after `ForgetSyncAuthStateDebugSetting` (since `ChangeToChinaSetting` starts with `"Toggle [...]`